README.rst

```diff
@@ -0,0 +1,35 @@
+..
+    Copyright (C) 2023 Graz University of Technology.
+
+    invenio-global-search is free software; you can redistribute it and/or
+    modify it under the terms of the MIT License; see LICENSE file for more
+    details.
+
+
+==============
+ invenio-global-search
+==============
+
+.. image:: https://github.com/tu-graz-library/invenio-global-search/workflows/CI/badge.svg
+        :target: https://github.com/tu-graz-library/invenio-global-search/actions?query=workflow%3ACI
+
+.. image:: https://img.shields.io/github/tag/tu-graz-library/invenio-global-search.svg
+        :target: https://github.com/tu-graz-library/invenio-global-search/releases
+
+.. image:: https://img.shields.io/pypi/dm/invenio-global-search.svg
+        :target: https://pypi.python.org/pypi/invenio-global-search
+
+.. image:: https://img.shields.io/github/license/tu-graz-library/invenio-global-search.svg
+        :target: https://github.com/tu-graz-library/invenio-global-search/blob/master/LICENSE
+
+Dublin Core data model for InvenioRDM
```

Comment:
Documentation copied !

Reply:
what do you mean? that there are copy paste errors or that i shouldn't copy that part?

Reply:
Yeah it is an information. The following lines do not belong to this module.